pandas/_libs/tslibs/period.pyx

```diff
@@ -1024,9 +1027,6 @@ def periodarr_to_dt64arr(int64_t[:] periodarr, int freq):
 
     with nogil:
         for i in range(l):
-            if periodarr[i] == NPY_NAT:
-                out[i] = NPY_NAT
-                continue
             out[i] = period_ordinal_to_dt64(periodarr[i], freq)
```

Comment:
is this now handled inside ?

Reply:
yep, has been for a while.  This is orthogonal to the segfault thing.